src/main.js

```diff
@@ -170,12 +159,56 @@ function displayNewCover(newCover) {
     }; 
```

Comment:
This function is the one that I think we still have the most questions about. We used the ondblclick method, but are unsure if it's frowned upon to directly edit the html in this way. Would event.target be preferred in this instance? From our reading, it seems like it would accomplish the same end goal, but we didn't understand it enough to use it.